### DISCUSSION TITLE
Unity 스크립트 기본 IDE 설정

### DISCUSSION THREAD

**Discussion by nonoaa** [42StudyGame/RandomDiceDefense42-Client#18] [Q&A]
Rider를 기본 설정으로 바꾸고 싶은데 유니티를 껐다킬때마다 Visual Studio로 돌아오네요
혹시 따로 방법이 있을까요?

**Comment by giftof** [comment]
<img width="709" alt="스크린샷 2022-10-03 오후 5 13 18" src="https://user-images.githubusercontent.com/72724626/193530664-b4addee8-40f3-4fdb-b552-561b23a337d0.png">
이렇게 설정해도, vs로 돌아온다는 뜻이신가용???

**Reply by nonoaa** [reply]
네 유니티 껐다키면 다시 vs로 돌아옵니다.. 원래는 안돌아오는게 맞는건가요?

**Reply by giftof** [reply]
네. 저렇게 설정하면 저장이 됩니다만..

**Reply by giftof** [reply]
저부분 말고, 다른것들은 설정값 저장 되나요?
game 탭의 해상도를 변경한다거나..
UI 배치를 변경한다거나...

**Reply by nonoaa** [reply]
네 화면배치를 바꾸던가 해상도를 바꾼다던가 하는 다른 설정들은 문제없이 잘 저장되는데 에디터만 계속 초기화되네요

**Reply by giftof** [reply]
https://learn.unity.com/tutorial/set-your-default-script-editor-ide#

우선... 이걸로 해도 안되면... 다음에 만나서 함 보실까요?

**Reply by nonoaa** [reply]
제가 작업을 데스크탑으로 해서 못보여드릴거같네요.. 일단은 킬때마다 변경해보겠습니다
